src/Coinbase.php

```diff
@@ -134,6 +134,17 @@ public function getCharge($chargeId)
         return $this->makeRequest('get', "charges/{$chargeId}");
     }
 
+    /**
+     * Deletes an existing charge.
+     *
+     * @param  string  $chargeId
+     * @return array
+     */
+    public function deleteCharge($chargeId)
+    {
+        return $this->makeRequest('delete', "charges/{$chargeId}/cancel");
```

Comment:
According to [the docs](https://docs.cloud.coinbase.com/commerce/reference/cancelcharge) this is a `POST` request, not `DELETE`.

```suggestion
        return $this->makeRequest('post', "charges/{$chargeId}/cancel");
```